### PR TITLE
Adding aws-nodejs-ecma-script template

### LINKED ISSUE #4055: Adding template for AWS Nodejs using ECMAScript
# This is a Feature Proposal

## Description
Add the ability to create function template for aws nodejs using ECMAScript:
`serverless create --template aws-nodejs-ecma-script --path awesome-es-project`

### PR DESCRIPTION
**Pull request by franciscocpg**
## What did you implement:

Closes #4055

## How did you implement it:
Adding templates in `lib/plugins/create/templates/aws-nodejs-ecma-script`

## How can we verify it:
Checkout and install this PR branch and then
`bin/serverless create --template aws-nodejs-ecma-script --path awesome-es-project`

## Todos:

- [x] Write tests
- [x] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config / commands / resources
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: lib/plugins/create/templates/aws-nodejs-ecma-script/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - serverless-webpack
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+
+functions:
+# Example without LAMBDA-PROXY integration
+# Invoking locally:
+# sls webpack invoke -f first
```
Comment (HyperBrain) [inline lib/plugins/create/templates/aws-nodejs-ecma-script/serverless.yml]: This sould be changed to `serverless invoke local` as soon as serverless-webpack v3 has been released, together with the package.json entry. Best would be in a follow up PR at a later time. For now it's ok.

Comment (franciscocpg) [inline lib/plugins/create/templates/aws-nodejs-ecma-script/serverless.yml]: @HyperBrain 
After serverless-webpack v3 maybe we could just remove this comment.

**Comment by eahefnawy** [comment]
@franciscocpg thanks for that 👍 ... could you add this template to the list of supported templates in the create command docs?

also, I don't know much about TypeScript, but is webpack a requirement? We try to keep all the core template focused and minimal 😊

**Comment by franciscocpg** [comment]
@eahefnawy 
Yes. Just to make sure, are you talking about [this section](https://github.com/serverless/serverless/blob/master/docs/providers/aws/cli-reference/create.md#available-templates)?

**Comment by pmuens** [comment]
@franciscocpg jumping in for @eahefnawy here.

Yes, correct! That section should be update. Best would be if you could do a global search for `aws-python3` (that's another template). This should show you all the places where the template should be added.

Thanks in advance! And let us know if you need any additional help here 😊

**Comment by franciscocpg** [comment]
Done @pmuens

**Pull request merged**